core/jazz_ui/e2e/08-multiaccountservices/secondwestservices.e2e-spec.ts

```diff
@@ -0,0 +1,443 @@
+  expect(jazzServices_po.getRepo().getText()).toEqual('Repository');
+  browser.wait(EC.visibilityOf(jazzServices_po.getRepository()), Common.timeOutHigh);
+  jazzServices_po.getRepository().click();
+  browser.sleep(Common.miniWait);
+ });
+
+ it('Create the Test Branch for second account with west region', () => {
+  browser.getAllWindowHandles().then(function (handles) {
+   browser.sleep(Common.microWait);
+   var min = 11;
+   var max = 99;
+   var randomNum = Math.floor(Math.random() * (max - min + 1)) + min;
+   var test = 'TEST' + randomNum;
+   browser.switchTo().window(handles[1]).then(function () {
+    browser.sleep(Common.microWait);
+
+    var some_name = browser.getTitle().then(function (webpagetitle) {
+     if (webpagetitle === 'Sign in · GitLab') {
+      expect(webpagetitle).toEqual('Sign in · GitLab');
+      jazzServices_po.gitUsername().sendKeys(Common.config.SCM_USERNAME).then(null, function (err) {
+       console.log(err.name);
+      });
+      browser.sleep(Common.microWait);
+      jazzServices_po.gitPassword().sendKeys(Common.config.SCM_PASSWORD).then(null, function (err) {
+       console.log(err.name);
+      });
+      browser.sleep(Common.microWait);
+      jazzServices_po.gitLogin().click().then(null, function (err) {
+       console.log(err.name);
+      });
+      browser.sleep(Common.microWait);
+      jazzServices_po.drpGitBranchType().click().then(null, function (err) {
+       console.log(err.name);
+      });
+      browser.sleep(Common.microWait);
+      jazzServices_po.selectGitBranchType().click().then(null, function (err) {
+       console.log(err.name);
+      });
+      browser.sleep(Common.microWait);
+      jazzServices_po.gitBranchName().sendKeys(test).then(null, function (err) {
+       console.log(err.name);
+      });
+      browser.sleep(Common.microWait);
+      jazzServices_po.btnGitCreateBranch().click().then(null, function (err) {
+       console.log(err.name);
+      });
+      browser.sleep(Common.microWait);
+      jazzServices_po.getGitLogoutIcon().click().then(null, function (err) {
+       console.log(err.name);
+      });
+      browser.sleep(Common.microWait);
+      jazzServices_po.getGitLogout().click().then(null, function (err) {
+       console.log(err.name);
+       flag = 0;
+       browser.sleep(Common.longWait);
+       browser.close();
+      });
+      browser.sleep(Common.microWait);
+      browser.close();
+     }
+     else {
+      expect(webpagetitle).not.toEqual('Sign in · GitLab');
+      jazzServices_po.bitUsername().sendKeys(Common.config.SCM_USERNAME).then(null, function (err) {
+       console.log(err.name);
+      });
+      browser.sleep(Common.microWait);
+      jazzServices_po.bitPassword().sendKeys(Common.config.SCM_PASSWORD).then(null, function (err) {
+       console.log(err.name);
+      });
+      browser.sleep(Common.microWait);
+      jazzServices_po.bitLogin().click().then(null, function (err) {
+       console.log(err.name);
```

Comment:
format error messages properly with more context on the error being printed.

Reply:
done